doc/release-notes.md

```diff
@@ -0,0 +1,400 @@
+Release Notes for BCH Unlimited 1.8.0
+======================================================
+
+BCH Unlimited version 1.8.0 is now available from:
+
+  <https://bitcoinunlimited.info/download>
+
+Please report bugs using the issue tracker at github:
+
+  <https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues>
+
+This is a major release of Bitcoin Unlimited compatible with the upcoming protocol upgrade of the Bitcoin Cash network. You could find
+May 15th, 2020 upgrade specifications here:
+
+- https://gitlab.com/bitcoin-cash-node/bchn-sw/bitcoincash-upgrade-specifications/-/blob/master/spec/2020-05-15-upgrade.md
+
+The following is a list of the previuos network upgrades specifications:
+
+- https://gitlab.com/bitcoin-cash-node/bchn-sw/bitcoincash-upgrade-specifications/-/blob/master/spec/uahf-technical-spec.md (Aug 1st '17, ver 1.1.0.0)
+- https://gitlab.com/bitcoin-cash-node/bchn-sw/bitcoincash-upgrade-specifications/-/blob/master/spec/nov-13-hardfork-spec.md (Nov 13th '17, ver 1.1.2.0)
+- https://gitlab.com/bitcoin-cash-node/bchn-sw/bitcoincash-upgrade-specifications/-/blob/master/spec/may-2018-hardfork.md (May 15th '18, ver 1.3.0.0, 1.3.0.1, 1.4.0.0)
+- https://gitlab.com/bitcoin-cash-node/bchn-sw/bitcoincash-upgrade-specifications/-/blob/master/spec/2018-nov-upgrade.md (Nov 15th '18, ver 1.5.0.0, 1.5.0.1, 1.5.0.2, 1.5.1.0)
+- https://gitlab.com/bitcoin-cash-node/bchn-sw/bitcoincash-upgrade-specifications/-/blob/master/spec/2019-05-15-upgrade.md (May 15th '19, ver 1.6.0.0)
+
+Upgrading
+---------
+
+If you are running an older version, shut it down. Wait until it has completely
+shut down (which might take a few minutes for older versions), then run the
+installer (on Windows) or just copy over /Applications/Bitcoin-Qt (on Mac) or
+bitcoind/bitcoin-qt (on Linux).
+
+Upgrading from a version lower than 1.7.0.0 would probably lead your client to be stuck
+on a minority chain and need some manual intervention to make so that it follow the majority
+chain after the upgrade. For more detail please look at `reconsidermostworkchain` RPC commands.
+
```

Comment:
This paragraph should no longer be needed since the reconsidermostworkchain would be automatic now.

Reply:
Great observation, will fix